docs/howto/deploy.md

```diff
@@ -84,23 +84,63 @@ export AZURE_OPERATOR_KEYVAULT=OperatorSecretKeyVault
 
 You can choose to use either Service Principals or Managed Identity for authentication.
 
-*Service Principal authentication* - if you choose to use Service Principal authentication, set these environment variables.
+#### Service Principal authentication
+If you choose to use Service Principal authentication, set these environment variables.
 
 ```shell
 export AZURE_CLIENT_ID=xxxxxxx
 export AZURE_CLIENT_SECRET=aaaaaaa
 ```
 
-*Managed Identity authentication* - if you choose to use Managed Identity, set the below environment variable and then perform the steps listed [here](managedidentity.md).
+#### Managed Identity authentication
+If you choose to use Managed Identity, set the below environment variables and then perform the steps listed [here](managedidentity.md).
 
 ```shell
+export AZURE_CLIENT_ID=xxxxxxx
 export AZURE_USE_MI=1
 ```
 
-> [!NOTE]
-> Use only one of the above.
+Before we can use Managed Identity authentication we need to install [aad-pod-identity](https://github.com/Azure/aad-pod-identity).
+
+*Installing AAD Pod Identity and registering an identity*
+
+1. Install [aad-pod-identity](https://github.com/Azure/aad-pod-identity).
+
+  ```shell
+  make install-aad-pod-identity
+  ```
+
+2. Apply the AzureIdentity and Binding manifests. This binds the identity to the Azure Service Operator. 
+Where a particular `resourceID` or `clientID` is referenced in the template below, ensure that you replace it with your Managed Identity `resourceID` and `clientID`.
+If you used the script mentioned [here](managedidentity.md), this should be automatically done for you.                                                                                                         
```

Comment:
Should the bash shell stuff below have specific subscription and identities removed, to encourage people to fill out their own?

Reply:
I'm not really aware of a way to templatize this and still leave it "clean" -- but what I can do at least is change the subscription ID to all 0's.

Reply:
Actually... I can just do `<your-x-here>`, maybe that's cleaner? I'll do that.